pkgs/misc/screensavers/electricsheep/default.nix

```diff
@@ -1,22 +1,51 @@
-{stdenv, fetchurl, pkgconfig, expat, zlib, libpng, libjpeg, xorg}:
+{ stdenv,
+  fetchgit,
+  pkgs,
+  #autoreconfHook
+}:
 
 stdenv.mkDerivation rec {
-  name = "electricsheep-2.6.8";
+  name = "${pname}-${version}";
+  pname = "electricsheep";
+  version = "2.7b33-${stdenv.lib.strings.substring 0 7 rev}";
+  rev = "598d93d90573b69559463567540aac8bc8a5b3f3";
   
-  src = fetchurl {
-    url = "http://electricsheep.org/${name}.tar.gz";
-    sha256 = "1flqcqfs75wg74hr5w85n6w8b26l4qrpwzi7fzylnry67yzf94y5";
+  src = fetchgit {
```

Comment:
I'd use `fetchFromGitHub` here and in the other one, which is quite a bit more efficient, especially for big repos.